stable/newrelic-infrastructure/templates/_helpers.tpl

```diff
@@ -0,0 +1,30 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "newrelic-infra.fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if ne $name .Release.Name -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s" $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+
+{{/* Generate basic labels */}}
+{{- define "newrelic-infra.labels" }}
+app: {{ template "name" . }}
+heritage: {{.Release.Service }}
+release: {{.Release.Name }}
```

Comment:
Why not include the `chart` label here as well?

Reply:
I was advised on a previous PR (#1254) that the pods should have three labels but other resources can have four. I only put three here so I can use this helper everywhere, only adding the `chart` label when this isn't applied to a pod, if that makes sense? I was trying to maintain consistency with my other PR.

Reply:
This is how it's usually done:
* All four labels on resources
* Labels `app` and `release` only on pods